app/assets/javascripts/misc/i18n-mode.js

```diff
@@ -5,15 +5,14 @@ document.addEventListener('DOMContentLoaded', function() {
   const inputs = document.querySelectorAll('input[type="submit"]');
 
   if (inputs) {
-    for (var i = 0; i < inputs.length; i++) {
-      const input = inputs[i];
+    [].slice.call(inputs).forEach((input) => {
```

Comment:
Doh, you're right, I didn't expand the diff preview and missed that part, good catch, sorry for the confusion!